src/helpers.ts

```diff
@@ -0,0 +1,26 @@
+//@ts-nocheck
+import { Product } from "./pages/types";
+
+export const getScreenWidth = (): number => {
```

Comment:
You can use `window.innerWidth` and `window.innerHeight`. 
By the way, it's possible to subscribe to `resize` events. Take a look at the example here https://bobbyhadz.com/blog/react-get-window-width-height